modules/services/vegbilder.js

```diff
@@ -34,18 +34,9 @@ async function fetchAvailableLayers() {
 
   const urlForRequest = owsEndpoint + utilQsString(params);
   const response = await d3_xml(urlForRequest);
-  const xPathSelector = '/wfs:WFS_Capabilities/wfs:FeatureTypeList/wfs:FeatureType/wfs:Name';
   const regexMatcher = /^vegbilder_1_0:Vegbilder(?<image_type>_360)?_(?<year>\d{4})$/;
-  const NSResolver = response.createNSResolver(response);
-  const l = response.evaluate(
-    xPathSelector,
-    response,
-    NSResolver,
-    XPathResult.ANY_TYPE
-    );
-  let node;
   const availableLayers = [];
-  while ( (node = l.iterateNext()) !== null ) {
+  for (const node of response.querySelectorAll('FeatureType > Name')) {
```

Comment:
jsdom doesn't support [`document.evaluate`](https://developer.mozilla.org/en-US/docs/Web/API/Document/evaluate) perfectly, and tbh I think this is more readable than 10 lines of XPath wizardry